_docs/pdksync.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: PDKSYNC
+---
+
+## PDKSYNC
+
+### Introduction
+
+Pdksync is an efficient way to run a pdk update command against the various Puppet module repositories that you manage — keeping them up-to-date with the changes made to PDK. 
+The tool checks out a module, runs the most recent version of pdk update then proceeds to create the PR for you with the changes. 
+This is the automated tool for ensuring modules can quickly adopt changes introduced by updates to the PDK and PDK template. 
+
+Pdksync by default expects that your Puppet module repositories live on GitHub and will behave accordingly. It also supports GitLab as an alternative Git hosting platform. 
+
+### Step-by-step guide
+
+Running the tool itself is quite simple and can be called from a rake task.
+
+- Download a fork of the repo, which can be found [here][https://github.com/puppetlabs/pdksync]. 
+Or you can install via Rubygems, it can be found [here][https://rubygems.org/gems/pdksync]. 
+- Install gems by using `bundle install`. 
+- Ensure you have a GITHUB_TOKEN or GITLAB_TOKEN set in your env, if you don't add it by running `export GITHUB_TOKEN=<your github token>` or `export GITLAB_TOKEN=<access_token>` , this is required for authentication. 
+- Important - Manually edit the list contained in 'managed_modules.yml' to ensure it is correct with the modules you wish to update. 
+Please note this is critical as this tool will create PRs against the repos included in this list - you don't want to run this against a module you aren't familiar with. 
+Do not proceed to the next step without doing this. 
+- Run the rake task by using `bundle exec rake pdksync`. 
+
+![pdksync - flowchart](pdksyncFlowchart.jpg)
+
+### Using pdksync for mass updates
+
+`bundle exec rake git:clone_managed_modules`
+
+`bundle exec rake 'pdksync:run_a_command[touch dog]'`
+
+`bundle exec rake 'git:create_commit[removeboltgem, (maint) Remove bolt gem from sync.yaml file]'`
+
+`bundle exec rake 'git:push_and_create_pr[(MODULES-xxxx) Remove bolt gem from the modules]'`
+
+### Using scripts with pdksync
+
+Sometimes you will want to make many changes in each module. below is a simple bash script to execute 3 commands in each module
+
+`bundle exec rake 'pdksync:run_a_command[cp ../../yamlbthere.rb .]'`
+`bundle exec rake 'pdksync:run_a_command[ruby yamlbthere.rb Gemfile]'`
+`bundle exec rake 'pdksync:run_a_command[rm -f yamlbthere.rb]'`
+
+Executing ruby scripts is even more powerful. 
+
+Imagine the script below running against every module. 
+
+```
+require 'yaml'
+require 'pry'
```

Comment:
Just a suggestion to remove the `require 'pry'` from the demo script

Reply:
@sanfrancrisko Thank you